andela/ninjas-ah-backend
# #165412934 bookmark an article

#### What does this PR do?
Add endpoints to allow a user to bookmark an article
#### Description of Task to be completed?
Have the following endpoints working and accessible to an authenticated user
- `PATCH /api/v1/articles/:slug/bookmark`
- `DELETE /api/v1/articles/:slug/bookmark`
- `GET /api/v1/articles/bookmarked`
#### How should this be manually tested?
- After cloning the repo, cd into it and type `npm test`
- Using Postman, firstly make sure you signup then login to get the access token, secondly create an article, finally make sure you pass the access token in the header with the following key name: `access-token`
#### What are the relevant pivotal tracker stories?
[165412934](https://www.pivotaltracker.com/story/show/165412934)
